### PR TITLE
docs: Update quickstart.md

### PR DESCRIPTION
**Pull request by azeltov**
**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by azeltov** [comment]
docs: fix cli  for exporting kubeconfig  and new line

File: docs/tutorials/quickstart.md [hunk 1]
```diff
@@ -105,7 +105,8 @@ azuredeploy.json		ca.key				etcdclient.key			etcdserver.key
 Access the new cluster by using the kubeconfig generated for the cluster's location. This example used `westus2`, so the kubeconfig is located at `_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json`:
 
 ```sh
-$ KUBECONFIG=_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json kubectl cluster-info
+$export KUBECONFIG=_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json
```
Comment (jackfrancis) [inline docs/tutorials/quickstart.md:108]:
```suggestion
$ export KUBECONFIG=_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json 
```

File: docs/tutorials/quickstart.md [hunk 2]
```diff
@@ -105,7 +105,8 @@ azuredeploy.json		ca.key				etcdclient.key			etcdserver.key
 Access the new cluster by using the kubeconfig generated for the cluster's location. This example used `westus2`, so the kubeconfig is located at `_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json`:
 
 ```sh
-$ KUBECONFIG=_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json kubectl cluster-info
+$export KUBECONFIG=_output/contoso-apple-5f776b0d/kubeconfig/kubeconfig.westus2.json 
+kubectl cluster-info
```
Comment (jackfrancis) [inline docs/tutorials/quickstart.md:109]:
```suggestion
$ kubectl cluster-info
```

**Review by jackfrancis — DISMISSED** [review]

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**